src/node/desktop/src/main/application.ts

```diff
@@ -295,4 +297,17 @@ export class Application implements AppState {
       );
     }
   }
+
+  setDockMenu(){
+    if (process.platform === 'darwin'){
+      const  menuDock = Menu.buildFromTemplate([{
+        label: 'New RStudio Window',
```

Comment:
Now that there's string externalization set up, this should be added.

Reply:
That's true my bad

Reply:
Fixed on [6dafb5f](https://github.com/rstudio/rstudio/pull/11499/commits/6dafb5fa11e7d4763a121ada108df55558a6ed90)